### DISCUSSION TITLE
How to make the time of unfinished pomodoros go into statistics?

### DISCUSSION THREAD

**Discussion by paivadaniel** [focustimerhq/FocusTimer#685] [Q&A]
Hello, imagine that I work with 50-minute pomodoros, and in one session I completed 32 minutes, but I have to leave and I will have to interrupt it. How do I add these 32 minutes to my day's statistics? By simply pressing pause, this amount of time should be saved in the statistics. This actually happened to me, I pressed pause, and turned off the computer, however, when I went back and looked at the statistics, there was no amount of time in the statistics for that day, as it was the first pomodoro of the day.

**Comment by kamilprusko** [comment]
Looks like a bug. Will look into it.

Currently there is no way of modifying the history / stats.

**Comment by paivadaniel** [comment]
I didn't understand. What is the bug?

**Reply by kamilprusko** [reply]
> this amount of time should be saved in the statistics / there was no amount of time in the statistics for that day

Like you said - it missed the 32 minutes
